Tests/AK/TestTypeTraits.cpp

```diff
@@ -39,8 +39,8 @@
 #define EXPECT_VARIADIC_TRAIT_FALSE(trait, ...) \
     static_assert(!trait<__VA_ARGS__>)
 
-enum class Enummer : u8 {
-    Dummmy,
+enum class Enumer : u8 {
```

Comment:
Is this supposed to be read as "Enumer[ation]" (in which case your fix would be correct) or as "Enum" -> "Enummer" as in "Run" -> "Runner" (in which case your fix would be incorrect)?